html/janus.js

```diff
@@ -1772,6 +1787,20 @@ function Janus(gatewayCallbacks) {
 				pluginHandle.onremotestream(config.remoteStream);
 				if(event.track.onended)
 					return;
+				if(config.receiverTransform) {
+					var receiverStreams = null;
+					if(event.track.kind === "audio") {
+						// Audio isn't supported by Insertable Streams yet
```

Comment:
it is but you're stuck on an old version :-)
You could do feature detection but... it should be there really soon

Reply:
Thanks for the clarification! Knowing how Chrome packagers hate Linux, I'll probably have to wait a couple of weeks at the very least for that to land in my `google-chrome-unstable`... Does this mean I can check if `RTCRtpSender.prototype.createEncodedAudioStreams` exists too, or is there a different check? By the time this is merged, I'll make sure Janus addresses encrypted audio streams too.

Reply:
I hope that next weeks unstable goes to M84. 84.0.4108.0 is pretty good. And yes @ createEncodedAudioStreams